.github/workflows/release.yml

```
@@ -0,0 +1,44 @@
+
+on:
+  workflow_dispatch:
+
+jobs:
+  release:
+    name: Release Extension
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
```

actionlint (1.7.4)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)